src/redprl/syntax.sml

```diff
@@ -16,14 +16,18 @@ struct
      VAR of variable * sort
    (* the trivial realizer for equality, which is called 'axiom' in NuPRL. *)
    | AX
+   (* the trivial realizer for interalized equality, less than, etc. *)
```

Comment:
Also, can we switch these two so that `AX` is for programs and `TV` is of sort `TRIV`?